web_accounts_list.json

```diff
@@ -1532,7 +1521,7 @@
          "name" : "SEOClerks",
          "check_uri" : "https://www.seoclerks.com/user/{account}",
          "account_existence_code" : "200",
-         "account_existence_string" : " - SEOClerks</title>",
+         "account_existence_string" : "<div class=\"user-info container\">",
          "account_missing_string" : "User not found",

```

Comment:
Please take @bcoles suggestion on making this account_missing_string more unique. 

> However, the account_missing_string string should also be updated. Requests for a non-existent user redirect to the home page. I suggest using <title>SEO Marketplace or <!-- Product List--> or Showing page:.